workbench-core/swb-common-ui/next.config.js

```diff
@@ -0,0 +1,11 @@
+const withTM = require('next-transpile-modules')(['@cloudscape-design/components']);
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true
+};
+
+module.exports = {
+  ...withTM(nextConfig),
+  trailingSlash: true
```

Comment:
this is related to the way routing works in next, specifically done to make the static website export work, if we dont need routing on this project we can just remove this line